File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Search/BedGroup.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses.Search

    Public Class BedGroupAvailability

        <JsonProperty("links")>
        Public Property Links As BedGroupAvailabilityLink

        <JsonProperty("id")>
        Public Property BedGroupID As String

        <JsonProperty("description")>
        Public Property Description As String

        <JsonProperty("configuration")>
        Public Property BedConfigurations As New List(Of BedConfiguration)

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Search/BedGroup.cs
namespace iVectorOne.CSSuppliers.ExpediaRapid.SerializableClasses.Search
{
    using System.Collections.Generic;
    using Newtonsoft.Json;


    public class BedGroupAvailability
    {

        [JsonProperty("links")]
        public BedGroupAvailabilityLink Links { get; set; }

        [JsonProperty("id")]
        public string BedGroupID { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("configuration")]
        public List<BedConfiguration> BedConfigurations { get; set; } = new List<BedConfiguration>();

    }

}